PittAPI/sports.py

```diff
@@ -0,0 +1,133 @@
+#   - in the off-season, this will return info on the latest game played.
+#   input_parameter specifies which type of information to get depending on what is entered.
+#   - full_name is something like "Pittsburgh Panthers vs.  Penn State Nittany Lions"
+#   - short_name is something like "PITT vs. PSU"
+#   - season_name is something like "Regular Season", "Playoffs", "Pre-season", etc.
+#   - week is something like "Week 19"
+#   - court returns the location of the game being played.
+
+# get_basketball_standings simply returns a string resembling something like "14th in ACC"
+
+
+def get_basketball_record() -> str:
+    try:
+        record_summary = basketball_data['team']['record']['items'][0]['summary']
+
+    except KeyError:
+        record_summary = "There's no record right now."
+
+    return record_summary
+
+
+def get_next_basketball_game(input_parameter: str) -> str:
+
+    return_value = "Error: Input parameter"
+
+    if input_parameter == "full_name":
+        return_value = basketball_data['team']['nextEvent'][0]['name']
+    if input_parameter == "short_name":
+        return_value = basketball_data['team']['nextEvent'][0]['shortName']
+
+    if input_parameter == "season_name":
+        return_value = basketball_data['team']['nextEvent'][0]['seasonType']['name']
+
+    if input_parameter == "week":
+        return_value = basketball_data['team']['nextEvent'][0]['week']['text']
+
+    if input_parameter == "court":
+        return_value = basketball_data['team']['nextEvent'][0]['competitions'][0]['venue']['fullName']
+
+    return return_value
+
+
+def get_basketball_standings() -> str:
+    return_value = basketball_data['team']['standingSummary']
+    return return_value
+
+
+# end basketball methods
+
+
+# football methods
+# get_football_record() is a method designed to get the current record of the Men's basketball team.
+#   - During the off-season this data should return "There's no record right now" TODO: ADD THIS EXCEPTION CATCH
+
+# get_next_football_game(input_parameter: str) is a method to get specific info on the next matchup.
+#   - in the off-season, this will return info on the latest game played.
+# input_parameter specifies which type of information to get depending on what is entered.
+#   - full_name is something like "Pittsburgh Panthers vs.  Penn State Nittany Lions"
+#   - short_name is something like "PITT vs. PSU"
+#   - season_name is something like "Regular Season", "Playoffs", "Pre-season", etc.
+#   - week is something like "Week 19"
+#   - field returns the location of the game being played.
+
+# get_basketball_standings simply returns a string resembling something like "14th in ACC"
+
+
+def get_football_record() -> str:
+    record_summary = football_data['team']['record']['items'][0]['summary']
+    return record_summary
+
+
+def get_next_football_game(input_parameter: str) -> str:
+
+    return_value = "Error: Input parameter"
```

Comment:
So in python, this is not a very good way of handling an error. I have another suggestion that should get rid of this but to more properly handle an error let's observe this code example I have below.

```python
valid_parameters = ['full_name', 'short_name']

if input_parameter not in valid_parameters:
    throw ValueError("Not a valid parameter")
```

So notice a few details, one we have a set of valid options that we can do a quick check on, second we throw an exception to the user noting this incorrect. Now, this forces the behavior of the user to fix the bug in their code and makes sure on our end this function only excepts a particular domain of input.

Pro tips:
- Look into the Python standard library errors, they have much pretty much all you really need without the need to create more unless you need to be very specific or group a few errors together
- `assert` is a wonderful keyword where you could as do as an example `assert input_parameter in valid_parameters` because if this isn't true it throws an assertion error